Tests/DangerTests/GitHubTestResources/GitHubBot.swift

```diff
@@ -0,0 +1,23 @@
+
+public let GitHubBotJSON = """
```

Comment:
Can you please format this JSON

Reply:
I have indented it and started the commit (it is running pre-commit hook tests etc.). Hopefully it will be the desired format :) If not let me know!